test/main.go

```diff
@@ -0,0 +1,5 @@
+package main
+
+func main()  {
+	type a struct{} // unused
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
U1000: type `a` is unused (unused)